news/5118.hooks.rst

```diff
@@ -0,0 +1 @@
+Remove support for deprecated PyQt4 and PySide
```

Comment:
Please alos copy this file as `5126.hooks.rst` (which is for the pull-request)

Reply:
oh right, it should be the pr# not the issue number

Reply:
It should be *two* files: One with teh issue, and one with the pr. This will make an entry like 
> * Remove support for deprecated PyQt4 and PySide (#5118, #5126)